Java/E-montazysta/src/main/java/com/emontazysta/controller/AppUserController.java

```diff
@@ -46,11 +45,11 @@ public ResponseEntity<List<AppUserDto>> getAppUsers() {
 
     @PostMapping("/create")
```

Comment:
I think that we should use same mapping in every controller, so if we want to use POST at /create we should use it everywhere.

Reply:
Same in ClientController and CommentController